packages/app/src/specs/SpecNameDisplay.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div
```

Comment:
```suggestion
  <div
        :title="specFileName + specFileExtension"
```

Small accessibility win :-)

Reply:
If you add this, would be good to add a corresponding test as well